app/views/dashboard/protocols/_summary.html.haml

```diff
@@ -32,16 +32,17 @@
       = edit_protocol_button_display(protocol, permission_to_edit)
     .clearfix
   .panel-body#protocol-summary.text-left
-    - if RESEARCH_MASTER_ENABLED
-      .row
-        .col-sm-2
-          %label
-            = I18n.t('protocols.summary.research_master_id')
-        .col-sm-10
-          - if protocol.research_master_id.nil?
-            = I18n.t('protocols.summary.na_research_master_id')
-          - else
-            = protocol.research_master_id
+    - if protocol.is_study?
```

Comment:
Could do if protocol.is_study? && RESEARCH_MASTER_ENABLED just to be a little more space efficient